app/src/ui/history/merge-call-to-action.tsx

```diff
@@ -11,6 +11,11 @@ interface IMergeCallToActionProps {
   readonly dispatcher: Dispatcher
   readonly currentBranch: Branch
   readonly formState: ICompareBranch
+
+  /**
+   * Function to execute after a merge has been performed
```

Comment:
We've been referring to these as callbacks across the codebase - a function that is invoked by _something else_ at some point. Can we stick to that pattern here?